pkg/testutil/testutil.go

```diff
@@ -41,3 +42,29 @@ func CompileInTempDir(t testing.TB) (tmpDir string, execPath string) {
 	}
 	return
 }
+
+// ErrorExists is only used to increase readability of future tests
+func ErrorExists(err error) bool {
+	return err != nil
+}
+
+// Helper function for PrintError
+func craftPrintMsg(errExists bool, out string) string {
+	var msg bytes.Buffer
+
+	if errExists {
+		msg.WriteString("Error Status: exists\n")
+	} else {
+		msg.WriteString("Error Status: not exists\n")
+	}
+	msg.WriteString("Output:\n")
+	msg.WriteString(out)
+	return msg.String()
+}
+
+// PrintError provides a standard way to print out error message when a test case fails
```

Comment:
This seems highly specialized to a very few cases and I think you'd best stick with just putting the errors in your test.

Reply:
Fixed